src/server/graph-generation/metadataMapperJson.js

```diff
@@ -152,7 +152,7 @@ function getElementFromBioPax(biopaxFile, id) {
   }
 
   //Get element matching the id
-  let result = biopaxFile.filter(data => data['pathid'] === id);
+  let result = biopaxFile.filter(data => data['pathid'].includes(id));
```

Comment:
Also, maybe you could use lodash.get with a default value of empty string to show others that data['pathid'] is supposed to be a string.

https://lodash.com/docs/4.17.4#get
i.e
```js
const _ = require('lodash');

let result = biopaxFile.filter(data => _.get(data, 'pathid', '').includes(id));
```

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find